src/methods/getColumn.ts

```diff
@@ -0,0 +1,37 @@
+import BinaryTree from "./binaryTree"
+import { TreeNode } from "./binaryTree";
+
+interface GetColumn extends BinaryTree<number> {
+    getColumn(node: TreeNode<number>, column: number): number[]
+}
+
+let currentColumn: number;
+let getColumnResult: number[];
```

Comment:
You don't need any global variables. In this particular case, I would say you can implement without them at all, but even if you need it, you should encapsulate the functionality under the class (use private methods/properties).